lib/simplecov.rb

```diff
@@ -54,6 +54,42 @@ def start(profile = nil, &block)
       start_coverage_measurment
     end
 
+    #
+    # Collate a series of SimpleCov result files into a single SimpleCov output.
+    # You can optionally specify configuration with a block:
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"]
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"], 'rails' # using rails profile
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"] do
+    #     add_filter 'test'
+    #   end
+    #    OR
+    #   SimpleCov.collate Dir["simplecov-resultset-*/.resultset.json"], 'rails' do
+    #     add_filter 'test'
+    #   end
+    #
+    # Please check out the RDoc for SimpleCov::Configuration to find about available config options
+    #
+    def collate(result_filenames, profile = nil, &block)
```

Comment:
I like that we have a nice feature test covering this, I'm just wondering that for a rather complex feature (although it looks fairly simple in its implementation) if a unit test or 2 wouldn't make sense.

I was thinking like:
1. a simple merge
2. there's nothing to merge (just one file)
3. no files are matching


 What do you think?

Reply:
We have those now! :clap: